Ym2149Synth/SynthPatchStorage.h

```diff
@@ -20,6 +20,18 @@
  *
  */
 
+/*
+ * Check for Teensy LC
+ *
+ * LC has only 128 bytes of flash-emulated EEPROM
+ * so we lower the patch amount to 11 instead of 16
+ */
+#if defined (__MKL26Z64__)
+#define MAX_PATCHES 11
+#else
+#define MAX_PATCHES 16
+#endif
+
```

Comment:
I was referring to the MAX_PATCHES changing from 16 to 11, but the "default" patches array only initializes 11 of the 16 in this branch. Which is why I was suggesting wrapping a compile time condition on that silly array. Thanks for the info & contrib 2xAA!